book.rb

```diff
@@ -0,0 +1,14 @@
+require_relative 'item'
+class Book < item
+  attr_accessor :publisher, :cover_state
+
+  def initialize(id, name, author, genre, publish_date, publisher, cover_state)
+    super(id, name, genre, author, label, publish_date)
```

Comment:
- [ ] Please check the `author` which is not defined in the Parent Class. I think it is unnecessary to add to our project.